<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorrowReturn</class>
 <widget class="QDialog" name="BorrowReturn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>501</width>
     <height>211</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Author </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Copies</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Borrow / Return Page</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectlabel_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Book ID: </string>
   </property>
  </widget>
  <widget class="QLabel" name="selectlabel_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Book ID: </string>
   </property>
  </widget>
  <widget class="QLabel" name="selectlabel_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrow a Book: </string>
   </property>
  </widget>
  <widget class="QLabel" name="selectlabel_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Return a Book </string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>711</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Status: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="returnID">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>460</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="borrowID">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="borrowButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrow</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
